autocompleter/registry.py

```diff
@@ -35,6 +39,7 @@ def register(self, ac_name, provider, local_settings=None):
         if local_settings is None:
             local_settings = {}
         self._ac_provider_settings[combined_name] = local_settings
+        self._ac_settings[ac_name] = local_settings
```

Comment:
We def don't want to do this... local_settings is meant as an way to set a bunch of as provider settings all at once ... perhaps we should rename local_settings to ac_provider_settings or something so it's clear?